src/module-1/splitAndMerge/index.js

```diff
@@ -1 +1,10 @@
-export const splitAndMerge = (str = "", separator = "") => {};
+export const splitAndMerge = (str = "", separator = "") => {
+  let splitStr =  str.split(" ");
+  let result= [];
```

Comment:
```suggestion
  const result = [];
```  
please consider using `.map` array method here and write it directly to the result: `const resultWords = words.map(value => {...});